RecoEgamma/EgammaTools/plugins/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
   <use   name="FWCore/Framework"/>
   <use   name="PhysicsTools/MVAComputer"/>
   <use   name="PhysicsTools/MVATrainer"/>
+  <use   name="PhysicsTools/PatAlgos"/>
```

Comment:
Really would prefer not to add things for standard reco:: objects to the
modifyObjects() functions in the base. Which seems to be a requirement here.

On Wed, Jul 1, 2015 at 12:25 AM, Slava Krutelyov <notifications@github.com>
wrote:

> In RecoEgamma/EgammaTools/plugins/BuildFile.xml
> <https://github.com/cms-sw/cmssw/pull/9659#discussion_r33630995>:
>
> > @@ -3,6 +3,7 @@
> >    <use   name="FWCore/Framework"/>
> >    <use   name="PhysicsTools/MVAComputer"/>
> >    <use   name="PhysicsTools/MVATrainer"/>
> > +  <use   name="PhysicsTools/PatAlgos"/>
>
> if you move ModifyObjectValueBase to CommonTools subsystem (as suggested
> above, the issue goes away)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9659/files#r33630995>.
>